ros2/ament_setup.hpp.tpl

```diff
@@ -0,0 +1,14 @@
+#pragma once
+
+namespace {{namespace}} {
+
+/// Sets the AMENT_PREFIX_PATH to the install space of this package.
+///
+/// This function needs to be called prior to any attempt to use a plugin in this package.
+///
+/// @param append If true, the install space is appended to the existing AMENT_PREFIX_PATH.
+/// @note Using append=True can lead to unexpected behavior when another version of a
+/// plugin used in this package is already present in the AMENT_PREFIX_PATH.
```

Comment:
The documentation should be IMHO in the doc of the top-level Bazel macro. Folks most likely won't ever open this template. On the other hand, we'll eventually auto-generate docs for Bazel API.

Reply:
Added additional docs there - I don't think it hurts to repeat the warning in both places.